.restyled.yaml

```diff
@@ -65,6 +65,7 @@ exclude:
     - "third_party/bluez/repo/**/*"
     - "third_party/cirque/repo/**/*"
     - "third_party/nanopb/repo/**/*"
+    - "src/controller/python/chip/clusters/CHIPClusters.py" # generated file
```

Comment:
Or we could have the generation run the restyler, like it does for C++ and Java files?  Followup is fine for this.

Reply:
Assuming followup for now since this is already a large change with high chance of merge conflicts.

I created #8948 for the  followup.